tests/test_optim/test_optimizer/test_optimizer.py

```diff
@@ -713,3 +722,78 @@ def test_default_optimizer_constructor_custom_key(self):
                     for setting in settings:
                         assert param_groups[i][setting] == settings[
                             setting], f'{name} {setting}'
+
+
+@unittest.skipIf(
+    digit_version(TORCH_VERSION) < digit_version('1.8.0'),
+    reason='ZeRO needs Pytorch 1.8 or higher')
+class TestZeroOptimizer(MultiProcessTestCase):
+
+    def setUp(self) -> None:
+        if ZeroRedundancyOptimizer is None:
+            self.skipTest('ZeroRedundancyOptimizer is not available.')
```

Comment:
```suggestion
        if ZeroRedundancyOptimizer is None:
            self.skipTest('ZeroRedundancyOptimizer is not available.')
```

Is this line duplicated with 
```python
@unittest.skipIf(
    digit_version(TORCH_VERSION) < digit_version('1.8.0'),
    reason='ZeRO needs Pytorch 1.8 or higher')
```

Reply:
https://github.com/open-mmlab/mmengine/actions/runs/3134972777/jobs/5090129146#step:8:132
I found that importing `ZeroRedundancyOptimizer` failed in the Windows CPU CI with & `torch1.8.1`.
(Importing failure made `_ZeroRedundancyOptimizer` as object.)
So I added duplicated skip code.

I'll check it again.

Reply:
I found that it has another condition.
`torch.distributed.rpc` should be available. I removed duplicated lines, and clarified this condition.